packages/use-sync-external-store/src/__tests__/useSyncExternalStoreShared-test.js

```diff
@@ -242,27 +240,29 @@ describe('Shared useSyncExternalStore behavior (shim and built-in)', () => {
     expect(root).toMatchRenderedOutput('A0B0');
 
     // Update b but not a
-    act(() => {
+    await act(() => {
```

Comment:
Just to check my understanding of the current state of act - we need to use await here because we're using internal act right? External act wouldn't?

Reply:
Yeah that's right. I think we could remove from internal act by mocking the microtask queue (which I think Jest already does, maybe?), just haven't yet. External act works because it bypasses the microtask queue entirely.